### DISCUSSION TITLE
Issue installing Hubot with Yo generator

### DISCUSSION THREAD

**Discussion by lrosser** [hubotio/hubot#1602] [Q&A] [answered]
I am trying to setup Hubot for the first time.

I think I have Node.js and npm installed correctly:
![image](https://user-images.githubusercontent.com/38354790/38756720-0897cbd6-3f30-11e8-86d5-f7ab241aca91.png)

I am using Windows Powershell ISE as my terminal.
PowerShell version:
![image](https://user-images.githubusercontent.com/38354790/38756873-8f223ee8-3f30-11e8-85db-6a8263707643.png)

I run npm install -g yo generator-hubot
and get the following...

![image](https://user-images.githubusercontent.com/38354790/38756625-c871d862-3f2f-11e8-9adf-5bb6a5dd3739.png)

then I run yo hubot

and it seems to hang at this point

![image](https://user-images.githubusercontent.com/38354790/38756647-dca782a0-3f2f-11e8-858b-b0e44baf2384.png)

Has anyone seen this before? Any ideas on what I am missing?

Thanks in advance for any help w/ this issue.

**Comment by Jossis** [comment, marked as answer]
Had this problem, couldn't find any explanation or solution. So i decided to try the setup with WSL (windows subsystem linux). And it worked wonders without any issuses. (https://www.easyaslinux.com/tutorials/devops/how-to-setup-an-efficient-chatbot-by-integrating-hubot-with-slack/)
